repos/fdbt-site/src/components/TapPricingRow.tsx

```diff
@@ -0,0 +1,97 @@
+                                <span className="govuk-visually"></span>
+                            </label>
+                        </FormElementWrapper>
+                    </>
+                </FormGroupWrapper>
+            </div>
+            <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                <FormGroupWrapper errors={errors} errorIds={[`multi-tap-price-${index}`]}>
+                    <>
+                        <label className="govuk-label" htmlFor={`multi-tap-price-${index}`}>
+                            <span className="govuk-visually-hidden">{`Tap Price - Price ${index + 1}`}</span>
+                            <span aria-hidden>Price</span>
+                        </label>
+                        <span className="govuk-hint" id={`tap-price-hint-${index}`}>
+                            {index === 0 ? 'e.g. 2.99' : ''}
+                        </span>
+
+                        <div className="govuk-currency-input">
+                            <div className="govuk-currency-input__inner">
+                                <span className="govuk-currency-input__inner__unit">£</span>
+                                <FormElementWrapper
+                                    errors={errors}
+                                    errorId={`multi-tap-price-${index}`}
+                                    errorClass="govuk-input--error"
+                                    hideText
+                                    addFormGroupError={false}
+                                >
+                                    <input
+                                        className="govuk-input govuk-input--width-4 govuk-currency-input__inner__input"
+                                        name={`multiTapPriceInput${index}`}
+                                        data-non-numeric
+                                        type="text"
+                                        aria-describedby={`tap-price-hint-${index}`}
+                                        id={`multi-tap-price-${index}`}
+                                        defaultValue={userInput[index]?.tapPrice ?? ''}
+                                    />
+                                </FormElementWrapper>
+                            </div>
+                        </div>
+                    </>
+                </FormGroupWrapper>
+            </div>
+        </div>
+    </fieldset>
+);
+
+export const renderRows = (
+    numberOfRows: number,
+    errors: ErrorInfo[],
+    userInput: MultiTapPricing[] = [],
+): ReactElement[] => {
+    const elements: ReactElement[] = [];
+    for (let i = 0; i < numberOfRows; i += 1) {
+        elements.push(renderTable(i, errors, userInput));
+    }
+    return elements;
+};
+
+const TapPricingRow = ({ numberOfTapsToDisplay, errors, userInput = [] }: TapPricingRowProps): ReactElement => {
+    return (
+        <div className={errors ? 'govuk-form-group--error' : ''}>
```

Comment:
```suggestion
        <div className={errors.length > 0 ? 'govuk-form-group--error' : ''}>
```

Reply:
Although, looking at other pages where we have this table, we dont do this in the renderRows function. Can we remove this div entirely?